EIPS/eip-7425.md

```diff
@@ -0,0 +1,175 @@
+	    * - MUST have closed proposalNumber
+	    * - MUST have userDeposit greater than or equal to userWithdrawal
+	    * @param assets Amount being deposited
+	    * @param receiver Address of receiver
+	    * @param owner Address of token owner
+	    * @param proposal Number associated proposal
+	    */
+	    function proposalWithdraw(uint256 assets, address receiver, address owner, uint256 proposal)external virtual returns(uint256);
+	    /**
+	    * @dev Burn shares, receive 1 to 1 value of assets
+	    * - MUST have closed proposal number
+	    * - MUST have userDeposit greater than or equal to userWithdrawal
+	    * NOTE: using ERC-4626 redeem() will not account for proposalWithdrawal
+	    * @param shares Amount being deposited
+	    * @param receiver Address of receiver
+	    * @param owner Address of token owner
+	    * @param proposal Number associated proposal
+	    */
+	    function proposalRedeem(uint256 shares, address receiver, address owner, uint256 proposal) external virtual returns(uint256);
+	    /**
+	    * @dev Issue new proposal
+	    * - MUST create new proposal number
+	    * - MUST account for amount withdrawn 
+	    * @param token Address of ERC-20 token
+	    * @param amount Token amount being withdrawn
+	    * @param receiver Address of token recipient
+	    */
+	    function proposalOpen(address token, uint256 amount, address receiver) external virtual returns (uint256);
+	    /**
+	    * @dev Close an opened proposal
+	    * - MUST account for amount received
+	    * - MUST be a proposal that is less than or equal to current proposal
+	    * @param token Address of ERC-20 token
+	    * @param proposal Number of desired proposal to close
+	    * @param amount Number assets being received
+	    */
+	    function proposalClose(address token, uint256 proposal, uint256 amount) external virtual returns (bool);
+	    /**
+	    * @dev Optional accounting for tokens deposited by owner
+	    * - MUST be reserve owner
+	    * NOTE: No shares are issued, funds can not be redeemed. Only withdrawn from proposalOpen
+	    * @param token Address of ERC-20 token
+	    * @param sender Address of where tokens from
+	    * @param amount Number of assets being deposited
+	    */
+	    function depositReserve(address token, address sender, uint256 amount) external virtual;
+    }
+
+```
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+## Rationale
+
+The reserve is designed to be a basic implementation of the reserve interface. Other non specified conditions should be addressed on case by case instances. Reserves use [ERC-20](../EIPS/eip-20.md) for shares and [ERC-4626](../EIPS/eip-4626.md) for creations of shares within proposals. All accounting measures are designed to enforce audit practices based on use case. 
```

Comment:
This doesn't really explain _why_ technical decisions were made. This would be a good place to explain, for example, why you extend ERC-4626.

Reply:
I updated the rationale section to be more descriptive about why ERC-4626 is being implemented in the reserve.